README.md

```diff
@@ -14,6 +14,8 @@ Whichever TTL is reached first will be used by the controller to delete the name
 The controller will will only consider pods matching the `!ci.openshift.io/ttl.ignore` selector for soft TTL calculations, allowing
 non-matching pods to keep running while considering the namespace to be inactive.
 
+In addition, the time after `Pod` termination or last active marker where, if no other `Pod`s are `Pending` or `Running`, the namespace should be reaped. The last active marker is optional and stored as a RFC3339-formatted date in the `ci.openshift.io/active` annotation.
```

Comment:
This should be the description column for soft ttl in the table above

Reply:
oops, I didn't notice it.